CefSharp.Wpf/Internals/WpfExtensions.cs

```diff
@@ -43,6 +43,21 @@ public static CefEventFlags GetModifiers(this MouseEventArgs e)
             return modifiers;
         }
 
+        /// <summary>
+        /// Gets the modifiers.
+        /// </summary>
+        /// <param name="e">The <see cref="DragEventArgs"/> instance containing the event data.</param>
+        /// <returns>CefEventFlags.</returns>
+        public static CefEventFlags GetModifiers(this DragEventArgs e)
+        {
+            CefEventFlags modifiers = 0;
+
+            modifiers |= GetModifierKeys(modifiers);
```

Comment:
This can likely be simplified to simply return GetModifierKeys():

Reply:
You are right. My brain clearly wasn't awake enough anymore at that time :see_no_evil:. (Started from the mouse one above.) Pushed the cleanup :slightly_smiling_face:

Reply:
Thanks 👍